std/string.d

```diff
@@ -73,6 +73,7 @@ $(TR $(TDNW Miscellaneous)
     )
 )))
 
+
```

Comment:
On Thu, Dec 31, 2015 at 05:26:23PM -0800, Jack Stouffer wrote:
> unnecessary line space

I explained this in the opening comment too.

"In a few places, I also add a blank line. This should be irrelevant to phobos, but it gives my doc generator a natural place to insert a table of contents after a brief introduction. The first line in ddoc is supposed to be a summary. Some functions lack this. Some write it with the assumption that it will be read immediately preceding the rest of the comment, making them read nonsensically if extracted into said table of contents. I reword a few of these"